uefi-raw/src/lib.rs

```diff
@@ -24,3 +24,11 @@ mod status;
 
 pub use status::Status;
 pub use uguid::{guid, Guid};
+
+/// Physical memory address. This is always a 64-bit value, regardless
+/// of target platform.
+pub type PhysicalAddress = u64;
```

Comment:
I'm not sure how beneficial this is.

Reply:
This typedef is part of the spec: [`EFI_PHYSICAL_ADDRESS`](https://uefi.org/specs/UEFI/2.10/07_Services_Boot_Services.html#efi-boot-services-allocatepages)

It gets used in a fair number of function definitions in the spec, so I think it's good to have.